backend/tests/node_values_crud.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
Nice integration tests add this to documentation and instruction how to run it xD

Reply:
They are an executable file, just run them directly. Let's change them to separate `proper` tests

Reply:
They are fine, they will be useful for documentation ;P